bitnami/etcd/3.5/debian-11/rootfs/opt/bitnami/scripts/libetcd.sh

```diff
@@ -666,10 +666,6 @@ etcd_initialize() {
                 export ETCD_INITIAL_CLUSTER_STATE=existing
             elif ! is_healthy_etcd_cluster; then
                 warn "Cluster not responding!"
-            fi
-            member_id="$(get_member_id)"
```

Comment:
Don't remove this. Instead, you can add it below line 663:

        if [[ ${#initial_members[@]} -gt 1 ]]; then
            member_id="$(get_member_id)"

Reply:
got it, added back